### ISSUE TITLE
Jetpack App - Blogging Prompts: Learn More is not interactable by VoiceOver on Landscape via Create New

### ISSUE THREAD

**Issue by ovitrif** [wordpress-mobile/WordPress-iOS#19610]
### Expected behavior

Users with VoiceOver enabled should be able to interact with the buttons on the screen to `Learn More` about blogging prompts in any orientation, regardless of how they accessed this page.

### Actual behavior
If the user with VoiceOver enabled opens the page to `Learn More` about blogging prompts from the sheet to Create New post/page/story, it's not possible to interact with the buttons on the screen when their iPhone is in Landscape orientation.

### Steps to reproduce the behavior
1. Open the app
2. Rotate the device to Landscape
3. Enable VoiceOver (avoid opening the app directly with VoiceOver because it [results in a crash](https://github.com/wordpress-mobile/WordPress-iOS/issues/19609))
4. Tap the colored Add (FAB-style) button to create a new post/page/story
5. Tap the small question mark at the right the `Prompts` title of the `Create New` action sheet, to learn more about blogging prompts
6. Try to tap any of the buttons (`X`, `Try it now`, `Remind me`)
7. **Notice** the buttons behind are selected and announced by VoiceOver
   It's not possible to close this screen with VoiceOver.

https://user-images.githubusercontent.com/4588074/202215520-de3020d2-6aae-4bff-9ec4-24d848d38c84.mp4

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
